apps/web/lib/video/[uid]/getServerSideProps.ts

```diff
@@ -228,6 +228,7 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
       rediectAttendeeToOnExit: isOrganizer
         ? undefined
         : bookingObj.eventType?.calVideoSettings?.redirectUrlOnExit,
+      overrideName: Array.isArray(context.query.name) ? context.query.name[0] : context.query.name,
```

Comment:
Q: When is it an array and when is it not?
Q: Can you point me to where we add this search param? e.g., how do people end up in `/video/1234?name=BENNY` instead of `/video/1234` (without any search param)?

Reply:
1) This is just to make sure the type of overrideName is always `string` and not `string | string[]` as you can also pass array in query params

2) This was requested by a customer who wanted to add this search param when user clicks a cal video link on their website.